lib/tether/tether.go

```diff
@@ -584,14 +582,37 @@ func logConfig(config *ExecutorConfig) {
 	// just pretty print the json for now
 	log.Info("Loaded executor config")
 
-	if log.GetLevel() == log.DebugLevel && config.DebugLevel > 1 {
-		sink := map[string]string{}
-		extraconfig.Encode(extraconfig.MapSink(sink), config)
+	// figure out the keys to filter
+	keys := make(map[string]interface{})
+	for _, k := range extraconfig.CalculateKeys(
+		config,
+		[]string{
+			"Sessions.*.Cmd.Args",
+			"Sessions.*.Cmd.Args.*",
+			"Sessions.*.Cmd.Env",
+			"Sessions.*.Cmd.Env.*",
+		},
+		"",
+	) {
+		keys[k] = nil
+	}
 
-		for k, v := range sink {
-			log.Debugf("%s: %s", k, v)
-		}
+	sink := map[string]string{}
+	extraconfig.Encode(
+		func(k, v string) error {
+			if _, ok := keys[k]; !ok {
+				sink[k] = v
+			}
+
+			return nil
+		},
+		config,
+	)
+
+	for k, v := range sink {
```

Comment:
The logging is in there is affected by the log level in the `extraconfig` package, which could be different from what the tether is using.